svn-auth.php

```diff
@@ -27,6 +26,14 @@
 $svn_path = preg_replace('/'.preg_quote($SVNLocationPath, '/').'/', $SVNParentPath, $location_path);
 // for folder reads it queries each item with !svn/ver/{revision}/ inserted into the path
 $svn_path = preg_replace('/!svn\/ver\/\d+\//', '', $svn_path);
+$svn_path = preg_replace('/!svn\/rvr\/\d+\//', '', $svn_path);
+
+// special WebDAV URIs
+// https://svn.apache.org/repos/asf/subversion/trunk/notes/http-and-webdav/http-protocol-v2.txt
+if (str_ends_with($location_path, "/!svn/me")) {
```

Comment:
If this is supposed to be for creating transactions, should this additionally be limited to when the passed in property is considered to be a `write` permission?